realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -704,6 +706,16 @@ public boolean isClosed() {
      */
     public abstract RealmSchema getSchema();
 
+    /**
+     * TODO
+     * @return
+     */
+    @ObjectServer
+    public SubscriptionSet getSubscriptions() {
```

Comment:
It is throwing further down in the OsSharedRealmImplementation. It would probably be nicer to throw sooner, but that is hard to achieve with how the code is currently split.